packages/workspace/src/command-line/format.ts

```diff
@@ -17,35 +17,27 @@ import {
 import { appRootPath } from '@nrwl/workspace/src/utilities/app-root';
 import { readFileSync, writeFileSync } from 'fs-extra';
 import * as stripJsonComments from 'strip-json-comments';
-
-const PRETTIER_EXTENSIONS = [
-  'ts',
-  'js',
-  'tsx',
-  'jsx',
-  'scss',
-  'less',
-  'css',
-  'html',
-  'json',
-  'md',
-  'mdx',
-  'graphql',
-  'gql',
-  'yaml',
-  'yml',
-];
-
-const MATCH_ALL_PATTERN = `**/*.{${PRETTIER_EXTENSIONS.join(',')}}`;
+import * as prettier from 'prettier';
 
 const PRETTIER_PATH = require.resolve('prettier/bin-prettier');
 
-export function format(command: 'check' | 'write', args: yargs.Arguments) {
+export function format(
+  command: 'check' | 'write',
+  args: yargs.Arguments
+): void {
   const { nxArgs } = splitArgsIntoNxArgsAndOverrides(args, 'affected');
 
-  const patterns = getPatterns({ ...args, ...nxArgs } as any).map(
-    (p) => `"${p}"`
-  );
+  const supportedExtensions = prettier
+    .getSupportInfo()
+    .languages.flatMap((language) => language.extensions)
+    .filter((extension) => !!extension);
+  const matchAllPattern = `**/*{${supportedExtensions.join(',')}}`;
```

Comment:
Can this be `.`?

Reply:
What exactly? Do you mean a `.` in front of the `{`?

I didn't do it because Prettier already returns the extensions with a leading dot, so the pattern would be something like `**/*{.js,.ts}` which is correct as well, though the `.` is repeated in all variations. That's why I also did this change https://github.com/nrwl/nx/pull/5877/files#diff-7b2b6b0996bbc7e183a20d67c2d30a7b6e1be57d6cc320bb055c9ec126f97378L79-R75. I can definitely strip the leading `.` so we have the pattern exactly as before, but I'm not sure we gain anything from that.